sdk/storage/Azure.Storage.Blobs/tests/SessionRecords/BlobBaseClientTests/Ctor_ConnectionString_Sas.json

```diff
@@ -0,0 +1,195 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://jolovstorage.blob.core.windows.net/test-container-8b104eb3-d6b7-8ee2-28ca-817e1ba9ed4a?sv=2019-02-02\u0026ss=bfq\u0026srt=sco\u0026spr=https\u0026se=2019-11-28T06%3A24%3A12Z\u0026sp=rwdlacup\u0026sig=Sanitized\u0026restype=container",
```

Comment:
These test recordings are used when running the tests in Playback mode which is what happens during CI. It may make sense to sanitize out the account name, but it isn't really required. If we made a change like that, we would have to re-record all of our tests.
This [link ](https://github.com/Azure/azure-sdk-for-net/blob/ab2e85209e9d665b58653f4dd13b6c11eaa5bc0a/Documentation/Track2TestFramework.md)has more info about the test infrastructure.